src/global-settings.php

```diff
@@ -54,7 +54,14 @@ function __construct() {
 			 * Typography hooks
 			 */
 
-			add_action( 'wp_enqueue_scripts', array( $this, 'typography_add_global_styles' ) );
+			/**
+			 * Uses `after_setup_theme` instead of `wp_enqueue_scripts` here for handling
+			 * page templates. Make sure to run `typography_add_global_styles` first before
+			 * the `typograhy_detect_native_blocks`  method.
```

Comment:
I think this comment needs to be updated now.

```suggestion
			 * Use `after_setup_theme` to check early if there are global
			 * typography used the `typograhy_detect_native_blocks`  method.
```